### DISCUSSION TITLE
Welcome to toppra Discussions!

### DISCUSSION THREAD

**Discussion by hungpham2511** [hungpham2511/toppra#164] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MinRen86** [comment]
How to use TOPPRA(c++) with path, start joint velocity and end joint velocity ?

**Reply by jmirabel** [reply]
This comment is unrelated to this thread. Generally, when you have a question, you should check whether someone has already asked a similar question and, if not, you should open a new discussion. Following these rules helps others find answers to already asked questions and reduces the weight on the maintainer. Thanks.

Regarding your specific issue, you should have a look at the unit test. I think there are sufficient information to achieve what you want.
